modules/primer-avatars/lib/avatar-stack.scss

```diff
@@ -43,3 +43,140 @@
     }
   }
 }
+
+// Refactored, new avatar stack:
+
+.AvatarStack {
+  position: relative;
+  min-width: 26px;
+  height: 20px;
+
+  &.AvatarStack--2 {
+    min-width: 36px;
+  }
+
+  &.AvatarStack--3 {
+    min-width: 46px;
+  }
+}
+
+.AvatarStack-body {
+  position: absolute;
+  display: flex;
+  background: $bg-white;
+
+  .avatar {
+    position: relative;
+    z-index: 2;
+    display: flex;
```

Comment:
What does this declaration do? (Things seem fine without it)

Reply:
This gets rid of the weird white-space disparity caused by `inline-block` :)